ceagle/api_fake_data/fake_runbooks.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import datetime
+import random
+
+import flask
+
+from ceagle.api_fake_data import base
+
+
+def get_single_runbook():
+    return {
+        "_id": str(random.randint(1, 1000)),
+        "description": "Demo runbook description",
+        "name": "Demo runbook",
+        "type": "bash",
+        "runbook": "IyEvYmluL2Jhc2gKCmVjaG8gIkhlbGxvIFdvcmxkISIK"
+    }
+
+
+def get_single_run():
+    return {
+        "_id": str(random.randint(1, 1000)),
+        "date": datetime.datetime.now(),
```

Comment:
Also, key name "date" looks inconsistent with the value, since value is date+time, however key "date" usually is used for YYY-dd-mm only